app_dart/lib/src/request_handlers/update_discord_status.dart

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:cocoon_common/rpc_model.dart' as rpc_model;
+import 'package:googleapis/firestore/v1.dart' show Document, Value;
+import 'package:meta/meta.dart';
+
+import '../../cocoon_service.dart';
+import '../service/discord_service.dart' show DiscordService;
+
+@immutable
+base class UpdateDiscordStatus extends GetBuildStatus {
```

Comment:
nit: You can make this `final class`.

(The reason the linter told you to add base or final is because GetBuildStatus is base)

Reply:
final it is.